style.css

```diff
@@ -154,6 +156,9 @@ footer {
   background-color: #a9a9a9;
   color: #fff;
 }
+#sriCopy {
+  width: 50px;
```

Comment:
I think we're trying to avoid the pixel unit for sizing. Can you use rem, to be consistent with other platforms??